examples/demo_dlmbl/python/excercise_1.py

```diff
@@ -56,16 +59,25 @@
 
 print(len(list(dataset.positions())))
 
+
 # %% [markdown]
 """
 View images with matplotlib.
 
+The layout on the disk is: row/col/field/resolution/timepoint/channel/z/y/x.
+
+
 Note that labelling is not perfect,
 as some cells are not expressing the fluorophore.
 """
 
 # %%
-image = dataset["0/0/0/0"].numpy()
+
+row = "0"
+col = "0"
+field = "0"
+resolution = "0" # 0 is the highest resolution, 1 is 2x2 binned, 2 is 4x4 binned, etc.
```

Comment:
In this dataset we will only be providing the full resolution. Will this be misleading?

Reply:
We should clarify the data format. The extra level of heirarchy in HCS format stumps a few people. Please add more resolutions using `ndpyramid` (https://pypi.org/project/ndpyramid/) in this dataset. We won't load these, but having them in dataset will clarify the format further.

Reply:
`ndpyramid` seems abandoned and does not support OME-Zarr. Since this is a 2D dataset, I will just use the [iohub method](https://czbiohub-sf.github.io/iohub/main/api/ngff.html#iohub.ngff.Position.initialize_pyramid) with [averaging](https://scikit-image.org/docs/stable/api/skimage.transform.html#skimage.transform.downscale_local_mean).

Reply:
Yes! I didn't know that iohub now has support for pyramids too.

Reply:
Done in [update pyramid dataset](https://github.com/mehta-lab/viscy/pull/29/commits/f8e2991a5f6cabd70e942872e09059a3b2c0be0a)